components/SendMoneyToCollectiveBtn.js

```diff
@@ -92,22 +103,22 @@ class SendMoneyToCollectiveBtn extends React.Component {
   }
 }
 
-const paymentMethodsQuery = gql`
+const paymentMethodsQuery = gqlV2/* GraphQL */ `
   query SendMoneyToCollectivePaymentMethods($slug: String) {
-    Collective(slug: $slug) {
+    account(slug: $slug) {
       id
       paymentMethods(service: "opencollective") {
```

Comment:
Can we add `type: "collective"` to be more precise and explicit here.

Reply:
Yes, that makes things simpler. Thanks. I've also [closed the API PR](https://github.com/opencollective/opencollective-api/pull/6343) as with this that is no longer needed. 😄

Reply:
I was not proposing to remove `service: "opencollective"`, I was proposing to **add** `type: "collective"`.